### ISSUE TITLE
In reply to should open a pane with the threaded conversation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#96]
```
What steps will reproduce the problem?
1. Find a tweet that has a "in reply to" bottom left
2. Click on the apparently clickable link

What is the expected output? What do you see instead?

I expect that to open up a pane with the threaded conversation shown.
Instead, nothing happens.

What version of the product are you using? On what operating system?
I'm using 0.1.5 on Ubuntu 9.10.

Please provide any additional information below.

```

Original issue reported on code.google.com by `spe...@gmail.com` on 18 Feb 2010 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
it would be a great improvement ;)
```

Original comment by `mich.ba...@gmail.com` on 18 Feb 2010 at 3:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 18 Feb 2010 at 11:33
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 81fdadbe3907 revision.
```

Original comment by `tro...@gmail.com` on 19 Feb 2010 at 4:10
- Changed state: **Fixed**
